workspace/api/openfdd_bridge/routes/model_routes.py

```
@@ -202,6 +202,34 @@ def run_model_sparql(body: SparqlBody, _user: dict = Depends(require_user)) -> d
+@router.get("/fdd-query-presets/{preset_id}")
+def fdd_query_preset_run(preset_id: str, _user: dict = Depends(require_user)) -> dict:
+    from ..fdd_query_presets import run_fdd_preset
+
+    try:
+        return run_fdd_preset(preset_id)
+    except KeyError as exc:
+        raise HTTPException(status_code=404, detail=f"unknown preset: {preset_id}") from exc
+
+
+@router.get("/fdd-coverage")
+def fdd_coverage(_user: dict = Depends(require_user)) -> dict:
```

Ruff (0.15.15)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)